src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is running");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator is stopped");
```

Comment:
Similarly, the message in the `stopWork` method should indicate that it's an `Excavator` that has stopped working.